doozer/doozerlib/cli/images_health.py

```diff
@@ -18,6 +20,43 @@
 now_unix_ts = int(round(time.time() * 1000))  # millis since the epoch
 
 
+def convert_to_iso_format(unix_timestamp_ms):
+    # Convert milliseconds to seconds
+    timestamp_seconds = unix_timestamp_ms / 1000.0
+    # Create a UTC datetime object from the timestamp
+    dt = datetime.datetime.utcfromtimestamp(timestamp_seconds)
+    # Format the datetime object to ISO 8601 format without fractional seconds
+    return dt.replace(microsecond=0).isoformat() + 'Z'
+
+
+def generate_art_dash_history_link(dg_name, build_record, runtime):
```

Comment:
I think only group and dg_name are needed, other params seem not useful

Reply:
Also correct, wanted to make sure the right build is always found so used all available parameters. If we dont need it then (as you say) I will change it.

Reply:
we need multi-builds for history not only one task, right?

Reply:
ah okay, so the art-dash link should not lead to just that build failure but to history of the build, this makes sense now and also follows Joeps strawhat proposal in the ticket.